llvm/lib/Target/PowerPC/PPCFrameLowering.cpp

```diff
@@ -2399,6 +2405,43 @@ bool PPCFrameLowering::assignCalleeSavedSpillSlots(
   return AllSpilledToReg;
 }
 
+static void findContinuousLoadStore(const MachineFunction *MF,
```

Comment:
```suggestion
static bool findConsecutiveLoadStore(const MachineFunction *MF,
```
Return `true` if consecutive ld/std found.